diff -rc fltk-1.1.10/FL/Enumerations.H fltk-1.1.10.patched/FL/Enumerations.H
*** fltk-1.1.10/FL/Enumerations.H	2009-07-02 09:42:17.000000000 +0000
--- fltk-1.1.10.patched/FL/Enumerations.H	2010-02-15 19:45:57.000000000 +0000
***************
*** 182,187 ****
--- 182,191 ----
    _FL_GTK_THIN_UP_BOX,	_FL_GTK_THIN_DOWN_BOX,
    _FL_GTK_THIN_UP_FRAME,	_FL_GTK_THIN_DOWN_FRAME,
    _FL_GTK_ROUND_UP_BOX,	_FL_GTK_ROUND_DOWN_BOX,
+   _FL_GLEAM_UP_BOX,	_FL_GLEAM_DOWN_BOX,
+   _FL_GLEAM_UP_FRAME,	_FL_GLEAM_DOWN_FRAME,
+   _FL_GLEAM_THIN_UP_BOX,	_FL_GLEAM_THIN_DOWN_BOX,
+   _FL_GLEAM_ROUND_UP_BOX,	_FL_GLEAM_ROUND_DOWN_BOX,
    FL_FREE_BOXTYPE
  };
  extern FL_EXPORT Fl_Boxtype fl_define_FL_ROUND_UP_BOX();
***************
*** 228,233 ****
--- 232,247 ----
  #define FL_GTK_ROUND_UP_BOX (Fl_Boxtype)(fl_define_FL_GTK_UP_BOX()+8)
  #define FL_GTK_ROUND_DOWN_BOX (Fl_Boxtype)(fl_define_FL_GTK_UP_BOX()+9)
  
+ extern FL_EXPORT Fl_Boxtype fl_define_FL_GLEAM_UP_BOX();
+ #define FL_GLEAM_UP_BOX fl_define_FL_GLEAM_UP_BOX()
+ #define FL_GLEAM_DOWN_BOX (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+1)
+ #define FL_GLEAM_UP_FRAME (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+2)
+ #define FL_GLEAM_DOWN_FRAME (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+3)
+ #define FL_GLEAM_THIN_UP_BOX (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+4)
+ #define FL_GLEAM_THIN_DOWN_BOX (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+5)
+ #define FL_GLEAM_ROUND_UP_BOX (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+6)
+ #define FL_GLEAM_ROUND_DOWN_BOX (Fl_Boxtype)(fl_define_FL_GLEAM_UP_BOX()+7)
+ 
  // conversions of box types to other boxtypes:
  inline Fl_Boxtype fl_box(Fl_Boxtype b) {
    return (Fl_Boxtype)((b<FL_UP_BOX||b%4>1)?b:(b-2));
diff -rc fltk-1.1.10/fluid/alignment_panel.cxx fltk-1.1.10.patched/fluid/alignment_panel.cxx
*** fltk-1.1.10/fluid/alignment_panel.cxx	2009-07-02 09:42:17.000000000 +0000
--- fltk-1.1.10.patched/fluid/alignment_panel.cxx	2010-02-15 19:46:26.000000000 +0000
***************
*** 181,186 ****
--- 181,187 ----
   {"None", 0,  0, 0, 0, FL_NORMAL_LABEL, 0, 14, 0},
   {"Plastic", 0,  0, 0, 0, FL_NORMAL_LABEL, 0, 14, 0},
   {"GTK+", 0,  0, 0, 0, FL_NORMAL_LABEL, 0, 14, 0},
+  {"Gleam", 0,  0, 0, 0, FL_NORMAL_LABEL, 0, 14, 0},
   {0,0,0,0,0,0,0,0,0}
  };
  
diff -rc fltk-1.1.10/fluid/alignment_panel.fl fltk-1.1.10.patched/fluid/alignment_panel.fl
*** fltk-1.1.10/fluid/alignment_panel.fl	2007-05-10 12:06:31.000000000 +0000
--- fltk-1.1.10.patched/fluid/alignment_panel.fl	2010-02-15 19:47:17.000000000 +0000
***************
*** 177,182 ****
--- 177,186 ----
          label {GTK+}
          xywh {10 10 35 25}
        }
+       MenuItem {} {
+         label {Gleam}
+         xywh {20 20 35 25}
+       }
      }
      Fl_Group {} {
        label {Options:
Binary files fltk-1.1.10/fluid/alignment_panel.o and fltk-1.1.10.patched/fluid/alignment_panel.o differ
Binary files fltk-1.1.10/fluid/Fl_Function_Type.o and fltk-1.1.10.patched/fluid/Fl_Function_Type.o differ
Binary files fltk-1.1.10/fluid/Fl_Menu_Type.o and fltk-1.1.10.patched/fluid/Fl_Menu_Type.o differ
Binary files fltk-1.1.10/fluid/fluid and fltk-1.1.10.patched/fluid/fluid differ
diff -rc fltk-1.1.10/fluid/fluid.cxx fltk-1.1.10.patched/fluid/fluid.cxx
*** fltk-1.1.10/fluid/fluid.cxx	2009-04-27 15:50:29.000000000 +0000
--- fltk-1.1.10.patched/fluid/fluid.cxx	2010-02-15 19:48:00.000000000 +0000
***************
*** 1695,1700 ****
--- 1695,1703 ----
      case 3 : // GTK+
        Fl::scheme("gtk+");
        break;
+     case 4 : // Gleam
+       Fl::scheme("gleam");
+       break;
    }
  
    fluid_prefs.set("scheme", scheme_choice->value());
Binary files fltk-1.1.10/fluid/fluid.o and fltk-1.1.10.patched/fluid/fluid.o differ
Binary files fltk-1.1.10/fluid/fluid-shared and fltk-1.1.10.patched/fluid/fluid-shared differ
diff -rc fltk-1.1.10/fluid/Fl_Widget_Type.cxx fltk-1.1.10.patched/fluid/Fl_Widget_Type.cxx
*** fltk-1.1.10/fluid/Fl_Widget_Type.cxx	2009-06-28 22:09:25.000000000 +0000
--- fltk-1.1.10.patched/fluid/Fl_Widget_Type.cxx	2010-02-15 19:49:20.000000000 +0000
***************
*** 673,678 ****
--- 673,684 ----
  {"GTK_THIN_DOWN_BOX",0,0,(void *)FL_GTK_THIN_DOWN_BOX},
  {"GTK_ROUND_UP_BOX",0,0,(void *)FL_GTK_ROUND_UP_BOX},
  {"GTK_ROUND_DOWN_BOX",0,0,(void *)FL_GTK_ROUND_DOWN_BOX},
+ {"GLEAM_UP_BOX",0,0,(void *)FL_GLEAM_UP_BOX},
+ {"GLEAM_DOWN_BOX",0,0,(void *)FL_GLEAM_DOWN_BOX},
+ {"GLEAM_THIN_UP_BOX",0,0,(void *)FL_GLEAM_THIN_UP_BOX},
+ {"GLEAM_THIN_DOWN_BOX",0,0,(void *)FL_GLEAM_THIN_DOWN_BOX},
+ {"GLEAM_ROUND_UP_BOX",0,0,(void *)FL_GLEAM_ROUND_UP_BOX},
+ {"GLEAM_ROUND_DOWN_BOX",0,0,(void *)FL_GLEAM_ROUND_DOWN_BOX},
  {0},
  {"frames",0,0,0,FL_SUBMENU},
  {"UP_FRAME",0,0,(void *)FL_UP_FRAME},
***************
*** 691,696 ****
--- 697,704 ----
  {"GTK_DOWN_FRAME",0,0,(void *)FL_GTK_DOWN_FRAME},
  {"GTK_THIN_UP_FRAME",0,0,(void *)FL_GTK_THIN_UP_FRAME},
  {"GTK_THIN_DOWN_FRAME",0,0,(void *)FL_GTK_THIN_DOWN_FRAME},
+ {"GLEAM_UP_FRAME",0,0,(void *)FL_GLEAM_UP_FRAME},
+ {"GLEAM_DOWN_FRAME",0,0,(void *)FL_GLEAM_DOWN_FRAME},
  {0},
  {0}};
  
Binary files fltk-1.1.10/fluid/Fl_Widget_Type.o and fltk-1.1.10.patched/fluid/Fl_Widget_Type.o differ
Binary files fltk-1.1.10/lib/libfltk.a and fltk-1.1.10.patched/lib/libfltk.a differ
Binary files fltk-1.1.10/lib/libfltk_forms.a and fltk-1.1.10.patched/lib/libfltk_forms.a differ
Binary files fltk-1.1.10/lib/libfltk_gl.a and fltk-1.1.10.patched/lib/libfltk_gl.a differ
Binary files fltk-1.1.10/lib/libfltk_images.a and fltk-1.1.10.patched/lib/libfltk_images.a differ
diff -rc fltk-1.1.10/src/fl_boxtype.cxx fltk-1.1.10.patched/src/fl_boxtype.cxx
*** fltk-1.1.10/src/fl_boxtype.cxx	2006-10-03 02:35:12.000000000 +0000
--- fltk-1.1.10.patched/src/fl_boxtype.cxx	2010-02-15 19:50:01.000000000 +0000
***************
*** 249,254 ****
--- 249,262 ----
    {fl_down_box,		1,1,2,2,0}, // _FL_GTK_THIN_ROUND_DOWN_BOX,
    {fl_up_box,		2,2,4,4,0}, // _FL_GTK_ROUND_UP_BOX,
    {fl_down_box,		2,2,4,4,0}, // _FL_GTK_ROUND_DOWN_BOX,
+   {fl_up_box,		2,2,3,3,0}, // _FL_GLEAM_UP_BOX,
+   {fl_down_box,		2,2,3,3,0}, // _FL_GLEAM_DOWN_BOX,
+   {fl_up_frame,		2,2,3,3,0}, // _FL_GLEAM_UP_FRAME,
+   {fl_down_frame,	2,2,3,3,0}, // _FL_GLEAM_DOWN_FRAME,
+   {fl_up_box,		1,1,1,1,0}, // _FL_GLEAM_THIN_UP_BOX,
+   {fl_down_box,		1,1,1,1,0}, // _FL_GLEAM_THIN_DOWN_BOX,
+   {fl_up_box,		2,2,3,3,0}, // _FL_GLEAM_ROUND_UP_BOX,
+   {fl_down_box,		2,2,3,3,0}, // _FL_GLEAM_ROUND_DOWN_BOX,
    {fl_up_box,		3,3,6,6,0}, // FL_FREE_BOX+0
    {fl_down_box,		3,3,6,6,0}, // FL_FREE_BOX+1
    {fl_up_box,		3,3,6,6,0}, // FL_FREE_BOX+2
Binary files fltk-1.1.10/src/fl_boxtype.o and fltk-1.1.10.patched/src/fl_boxtype.o differ
Binary files fltk-1.1.10/src/Fl_File_Browser.o and fltk-1.1.10.patched/src/Fl_File_Browser.o differ
diff -rc fltk-1.1.10/src/Fl_get_system_colors.cxx fltk-1.1.10.patched/src/Fl_get_system_colors.cxx
*** fltk-1.1.10/src/Fl_get_system_colors.cxx	2007-02-20 17:02:41.000000000 +0000
--- fltk-1.1.10.patched/src/Fl_get_system_colors.cxx	2010-02-15 19:54:40.000000000 +0000
***************
*** 346,351 ****
--- 346,374 ----
  
      // Use slightly thinner scrollbars...
      Fl::scrollbar_size(15);
+     
+   } else if (scheme_ && !strcasecmp(scheme_, "gleam")) {
+      // Use a Gleam look-and-feel, similar to Clearlooks Glossy, made by Colin Jones
+      if (scheme_bg_) {
+        delete scheme_bg_;
+        scheme_bg_ = (Fl_Image *)0;
+      }
+ 
+      set_boxtype(FL_UP_FRAME,        FL_GLEAM_UP_FRAME);
+      set_boxtype(FL_DOWN_FRAME,      FL_GLEAM_DOWN_FRAME);
+      set_boxtype(FL_THIN_UP_FRAME,   FL_GLEAM_UP_FRAME);
+      set_boxtype(FL_THIN_DOWN_FRAME, FL_GLEAM_DOWN_FRAME);
+  
+      set_boxtype(FL_UP_BOX,          FL_GLEAM_UP_BOX);
+      set_boxtype(FL_DOWN_BOX,        FL_GLEAM_DOWN_BOX);
+      set_boxtype(FL_THIN_UP_BOX,     FL_GLEAM_THIN_UP_BOX);
+      set_boxtype(FL_THIN_DOWN_BOX,   FL_GLEAM_THIN_DOWN_BOX);
+      set_boxtype(_FL_ROUND_UP_BOX,   FL_GLEAM_ROUND_UP_BOX);
+      set_boxtype(_FL_ROUND_DOWN_BOX, FL_GLEAM_ROUND_DOWN_BOX);
+  
+      // Use slightly thinner scrollbars...
+      Fl::scrollbar_size(12);
+ 
    } else {
      // Use the standard FLTK look-n-feel...
      if (scheme_bg_) {
Binary files fltk-1.1.10/src/Fl_get_system_colors.o and fltk-1.1.10.patched/src/Fl_get_system_colors.o differ
Binary files fltk-1.1.10/src/Fl_GIF_Image.o and fltk-1.1.10.patched/src/Fl_GIF_Image.o differ
Only in fltk-1.1.10.patched/src: fl_gleam.cxx
Only in fltk-1.1.10.patched/src: fl_gleam.o
Binary files fltk-1.1.10/src/Fl_Menu.o and fltk-1.1.10.patched/src/Fl_Menu.o differ
Binary files fltk-1.1.10/src/gl_draw.o and fltk-1.1.10.patched/src/gl_draw.o differ
Binary files fltk-1.1.10/src/libfltk_gl.so and fltk-1.1.10.patched/src/libfltk_gl.so differ
Binary files fltk-1.1.10/src/libfltk_gl.so.1.1 and fltk-1.1.10.patched/src/libfltk_gl.so.1.1 differ
Binary files fltk-1.1.10/src/libfltk_images.so and fltk-1.1.10.patched/src/libfltk_images.so differ
Binary files fltk-1.1.10/src/libfltk_images.so.1.1 and fltk-1.1.10.patched/src/libfltk_images.so.1.1 differ
Binary files fltk-1.1.10/src/libfltk.so and fltk-1.1.10.patched/src/libfltk.so differ
Binary files fltk-1.1.10/src/libfltk.so.1.1 and fltk-1.1.10.patched/src/libfltk.so.1.1 differ
diff -rc fltk-1.1.10/src/makedepend fltk-1.1.10.patched/src/makedepend
*** fltk-1.1.10/src/makedepend	2007-02-10 14:31:12.000000000 +0000
--- fltk-1.1.10.patched/src/makedepend	2010-02-15 19:51:51.000000000 +0000
***************
*** 381,386 ****
--- 381,389 ----
    ../FL/Fl.H ../FL/Enumerations.H ../FL/Fl_Export.H ../FL/fl_draw.H \
    ../FL/x.H ../FL/Fl_Window.H ../FL/Fl_Group.H ../FL/Fl_Widget.H \
    Fl_Font.H fl_font_xft.cxx
+ fl_gleam.o: fl_gleam.cxx ../FL/Fl.H ../FL/Enumerations.H \
+   ../FL/Fl_Export.H ../FL/fl_draw.H flstring.h ../FL/Fl_Export.H \
+   ../config.h  
  fl_gtk.o: fl_gtk.cxx ../FL/Fl.H ../FL/Enumerations.H ../FL/Fl_Export.H \
    ../FL/fl_draw.H
  fl_labeltype.o: fl_labeltype.cxx ../FL/Fl.H ../FL/Enumerations.H \
diff -rc fltk-1.1.10/src/Makefile fltk-1.1.10.patched/src/Makefile
*** fltk-1.1.10/src/Makefile	2009-07-02 09:42:17.000000000 +0000
--- fltk-1.1.10.patched/src/Makefile	2010-02-15 19:52:45.000000000 +0000
***************
*** 133,138 ****
--- 133,139 ----
  	fl_engraved_label.cxx \
  	fl_file_dir.cxx \
  	fl_font.cxx \
+ 	fl_gleam.cxx \
  	fl_gtk.cxx \
  	fl_labeltype.cxx \
  	fl_line_style.cxx \
Binary files fltk-1.1.10/test/adjuster and fltk-1.1.10.patched/test/adjuster differ
Binary files fltk-1.1.10/test/arc and fltk-1.1.10.patched/test/arc differ
Binary files fltk-1.1.10/test/ask and fltk-1.1.10.patched/test/ask differ
Binary files fltk-1.1.10/test/bitmap and fltk-1.1.10.patched/test/bitmap differ
Binary files fltk-1.1.10/test/blocks and fltk-1.1.10.patched/test/blocks differ
Binary files fltk-1.1.10/test/boxtype and fltk-1.1.10.patched/test/boxtype differ
Binary files fltk-1.1.10/test/browser and fltk-1.1.10.patched/test/browser differ
Binary files fltk-1.1.10/test/button and fltk-1.1.10.patched/test/button differ
Binary files fltk-1.1.10/test/buttons and fltk-1.1.10.patched/test/buttons differ
Binary files fltk-1.1.10/test/checkers and fltk-1.1.10.patched/test/checkers differ
Binary files fltk-1.1.10/test/clock and fltk-1.1.10.patched/test/clock differ
Binary files fltk-1.1.10/test/colbrowser and fltk-1.1.10.patched/test/colbrowser differ
Binary files fltk-1.1.10/test/color_chooser and fltk-1.1.10.patched/test/color_chooser differ
Binary files fltk-1.1.10/test/cube and fltk-1.1.10.patched/test/cube differ
Binary files fltk-1.1.10/test/CubeView and fltk-1.1.10.patched/test/CubeView differ
Binary files fltk-1.1.10/test/cursor and fltk-1.1.10.patched/test/cursor differ
Binary files fltk-1.1.10/test/curve and fltk-1.1.10.patched/test/curve differ
Binary files fltk-1.1.10/test/demo and fltk-1.1.10.patched/test/demo differ
Binary files fltk-1.1.10/test/doublebuffer and fltk-1.1.10.patched/test/doublebuffer differ
Binary files fltk-1.1.10/test/editor and fltk-1.1.10.patched/test/editor differ
Binary files fltk-1.1.10/test/fast_slow and fltk-1.1.10.patched/test/fast_slow differ
Binary files fltk-1.1.10/test/file_chooser and fltk-1.1.10.patched/test/file_chooser differ
Binary files fltk-1.1.10/test/fonts and fltk-1.1.10.patched/test/fonts differ
Binary files fltk-1.1.10/test/forms and fltk-1.1.10.patched/test/forms differ
Binary files fltk-1.1.10/test/fractals and fltk-1.1.10.patched/test/fractals differ
Binary files fltk-1.1.10/test/fullscreen and fltk-1.1.10.patched/test/fullscreen differ
Binary files fltk-1.1.10/test/gl_overlay and fltk-1.1.10.patched/test/gl_overlay differ
Binary files fltk-1.1.10/test/glpuzzle and fltk-1.1.10.patched/test/glpuzzle differ
Binary files fltk-1.1.10/test/hello and fltk-1.1.10.patched/test/hello differ
Binary files fltk-1.1.10/test/help and fltk-1.1.10.patched/test/help differ
Binary files fltk-1.1.10/test/iconize and fltk-1.1.10.patched/test/iconize differ
Binary files fltk-1.1.10/test/image and fltk-1.1.10.patched/test/image differ
Binary files fltk-1.1.10/test/inactive and fltk-1.1.10.patched/test/inactive differ
Binary files fltk-1.1.10/test/input and fltk-1.1.10.patched/test/input differ
Binary files fltk-1.1.10/test/input_choice and fltk-1.1.10.patched/test/input_choice differ
Binary files fltk-1.1.10/test/keyboard and fltk-1.1.10.patched/test/keyboard differ
Binary files fltk-1.1.10/test/label and fltk-1.1.10.patched/test/label differ
Binary files fltk-1.1.10/test/line_style and fltk-1.1.10.patched/test/line_style differ
Binary files fltk-1.1.10/test/mandelbrot and fltk-1.1.10.patched/test/mandelbrot differ
Binary files fltk-1.1.10/test/menubar and fltk-1.1.10.patched/test/menubar differ
Binary files fltk-1.1.10/test/message and fltk-1.1.10.patched/test/message differ
Binary files fltk-1.1.10/test/minimum and fltk-1.1.10.patched/test/minimum differ
Binary files fltk-1.1.10/test/navigation and fltk-1.1.10.patched/test/navigation differ
Binary files fltk-1.1.10/test/output and fltk-1.1.10.patched/test/output differ
Binary files fltk-1.1.10/test/overlay and fltk-1.1.10.patched/test/overlay differ
Binary files fltk-1.1.10/test/pack and fltk-1.1.10.patched/test/pack differ
Binary files fltk-1.1.10/test/pixmap and fltk-1.1.10.patched/test/pixmap differ
Binary files fltk-1.1.10/test/pixmap_browser and fltk-1.1.10.patched/test/pixmap_browser differ
Binary files fltk-1.1.10/test/preferences and fltk-1.1.10.patched/test/preferences differ
Binary files fltk-1.1.10/test/radio and fltk-1.1.10.patched/test/radio differ
Binary files fltk-1.1.10/test/resize and fltk-1.1.10.patched/test/resize differ
Binary files fltk-1.1.10/test/resizebox and fltk-1.1.10.patched/test/resizebox differ
Binary files fltk-1.1.10/test/resizebox.o and fltk-1.1.10.patched/test/resizebox.o differ
Binary files fltk-1.1.10/test/scroll and fltk-1.1.10.patched/test/scroll differ
Binary files fltk-1.1.10/test/shape and fltk-1.1.10.patched/test/shape differ
Binary files fltk-1.1.10/test/subwindow and fltk-1.1.10.patched/test/subwindow differ
Binary files fltk-1.1.10/test/sudoku and fltk-1.1.10.patched/test/sudoku differ
Binary files fltk-1.1.10/test/symbols and fltk-1.1.10.patched/test/symbols differ
Binary files fltk-1.1.10/test/tabs and fltk-1.1.10.patched/test/tabs differ
Binary files fltk-1.1.10/test/threads and fltk-1.1.10.patched/test/threads differ
Binary files fltk-1.1.10/test/tile and fltk-1.1.10.patched/test/tile differ
Binary files fltk-1.1.10/test/tiled_image and fltk-1.1.10.patched/test/tiled_image differ
Binary files fltk-1.1.10/test/unittests and fltk-1.1.10.patched/test/unittests differ
Binary files fltk-1.1.10/test/valuators and fltk-1.1.10.patched/test/valuators differ
